packages/amplify-graphql-api-construct/src/amplify-graphql-api.ts

```diff
@@ -87,6 +87,8 @@ export class AmplifyGraphqlApi extends Construct {
    */
   public readonly resources: AmplifyGraphqlApiResources;
 
+  public readonly stack: Stack;
```

Comment:
Add doc comment explaining what stack this refers to

Reply:
it has been added 👍